public/admin/index.html

```diff
@@ -48,14 +48,15 @@ <h3>Existing URLs</h3>
 
     <!-- Firebase -->
 
-    <script src="https://www.gstatic.com/firebasejs/4.9.1/firebase.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/8.10.0/firebase.js"></script>
     <script src="firebaseConfig.js"></script>
     <script>
         // Initialize Firebase
         firebase.initializeApp(firebaseConfig);
+		firebase.auth().useEmulator("http://localhost:9099");
```

Comment:
ALERT ALERT, unwanted dev emulator - can cause to broke production!

Reply:
oh, my bad, forgot to remove it